sdk/metric/batcher/defaultkeys/defaultkeys_test.go

```diff
@@ -56,7 +56,7 @@ func TestGroupingStateless(t *testing.T) {
 	// Repeat for {counter,lastvalue}.{1,2}.
 	// Output lastvalue should have only the "G=H" and "G=" keys.
 	// Output counter should have only the "C=D" and "C=" keys.
-	require.EqualValues(t, map[string]int64{
+	require.EqualValues(t, map[string]float64{
```

Comment:
Why this change? I don't get it.

Reply:
I added a new test in `sdk/metric` because I had let a bug slip through and only the OTLP end-to-end test recently introduced actually tested that the real SDK would call the observers. The new test uses this helper.